projects/BEVFusion/bevfusion/bevfusion.py

```diff
@@ -239,4 +239,19 @@ def extract_feat(
     def loss(self, batch_inputs_dict: Dict[str, Optional[Tensor]],
              batch_data_samples: List[Det3DDataSample],
              **kwargs) -> List[Det3DDataSample]:
-        pass
+        batch_input_metas = [item.metainfo for item in batch_data_samples]
+        feats = self.extract_feat(batch_inputs_dict, batch_input_metas)
```

Comment:
The `GlobalRotScaleTrans` and `RandomFlip3D` in mmdet have not recorded the `lidar_aug_matrix` and `img_aug_matrix` like BEVFusion. We should use these aug matrices in image view transformation in `extrack_img_feat` function.